meetings/2019-06-17.md

```diff
@@ -0,0 +1,103 @@
+  * LT: but it can happen that there are vulns with no known fix, so then the package maintainer will have to get back to us with an update
+  * RP: the H1 docs say that we will make the vuln live in 45 days, but he doesn’t think we do what it says, we avoid making them live
+  * MD: is that common?
+  * LT and RP: yes!
+  * LT: if they don’t respond, we will have to release the vuln, and it will have to have a `*`
+  * SR: seems consensus is we try to not release vulns where maintainer has not released a fix, but if they don’t , we have to release, and we do with a `*`
+
+* HackerOne's managed services [#516](https://github.com/nodejs/security-wg/issues/516)
+  * VT: H1 is doing triage, we can probably remove this from agenda
+  * RP: H1 is doing triage into a next queue that sec-wg has to triage
+  * SR: are enough people working on getting stuff moving in H1?
+  * LT: have about 23, its going slowly
+  * SR: do we need more people?
+  * LT: Ron has been doing lots, he is going to start, but adding people might not make it go faster
+  * RP: plans to start working on more this week
+  * SR: maybe we should report in every meeting current state?
+  * RP: agreed to own reporting
+  * LT: thinks its a good idea
+  * VT: taking off agenda
+
+* Reaching out to other projects of the OpenJS Foundation [#511](https://github.com/nodejs/security-wg/issues/511)
+  * VT: waiting for updates about state of openjs foundation
+  * MD: voting representatives known, some work moving forward, this is good time to open an issue in the openjs foundation to raise a suggestion
+  * VT: where?
+  * MD: probably good time to open an issue here: https://github.com/openjs-foundation/cross-project-council
+  * MD: what is it you think we should do? Using ecosystem triage for all projects?
+  * VT: yes, and suggest that all repos have security processes, and they involve the sec-wg’s triage
+  * MD: seems reasonable
+  * VT: invite them, and offer to remove some sec burden from the project’s maintainers
+
+* Attendance in the upcoming Collaborator Summit in May? [#510](https://github.com/nodejs/security-wg/issues/510)
+  * SR: remove from agenda, its already happened
+
+* Large-scale disclosure guidelines [#505](https://github.com/nodejs/security-wg/issues/505)
+  * N: its time to publish the list of packages using the buffer constructor, sorted by downloads
+  * SR: agree, this information is public, bad actors know everything they need to scrape npm packages to find vulns, lets make the info public so it can be used
+  * LT: perhaps start with a blog post describing the state, and that we are about to publish this information
+  * MD: will we be able to say what people should do?
+  * LT: yes, there are alternatives. Lets just make sure that there is no surprise
+  * MD: agree, as long as there is a solution.
+  * N: since 4.5 there has been alternative APIs available that fixes the memory disclosure problem. DoS can’t be fixed
```

Comment:
That's not exactly what I said. I'll update the google docs.

Reply:
Can you just post the text you want here, or use the "suggested fix feature", or push a fixup to my branch? I'd have to somehow diff the google or reformat the whole thing as markdown, again.

Reply:
ping, @ChALkeR

Reply:
@sam-github Sorry, missed that. That was a one-line change, here it is:
```suggestion
  * N: since 4.5 there has been alternative APIs available that fixes the issues on user side. Fixes in newer Node versions solved the memory disclosure problem in buffer constructor, but DoS can’t be fixed without fixes on user side (i.e. migrating to the new APIs introduced in 4.5).
```